runtime/src/chpl-comm.c

```diff
@@ -44,6 +44,7 @@
 
 int32_t          chpl_nodeID = -1;
 int32_t          chpl_numNodes = -1;
+const char       *chpl_comm_oob = "unknown";
```

Comment:
Should this be a general comm layer things for now, or ofi specific?

Reply:
Right now it's only used in ofi so maybe it should have `ofi` in the name.

Reply:
Instead of putting ofi in the name, I was wondering if this can just be implemented in an ofi header/source file instead of the general `chpl-comm.{c,h}`

Reply:
Moved to `ofi`.